httpcore5-h2/src/main/java/org/apache/hc/core5/http2/nio/AsyncPingHandler.java

```diff
@@ -49,6 +49,8 @@
      * Triggered to signal receipt of a ping response message.
      *
      * @param feedback the ping message feedback.
+     * @throws IOException in case of an I/O error
```

Comment:
@arturobernalg I honestly think this is just not worth it. `@throws IOException in case of an I/O error` adds no value to the javadocs. I am not sure what I am to do about this PR.

Reply:
hi @ok2c maybe you're right. Any advice to how proceed now? I'm honestly miss the javadoc in the above class and don't think it's important give all information that we can to the user's.

Reply:
@arturobernalg Are these all javadoc fixes you have in mind or are you planning on adding more at some point? Preferably I would like to avoid multiple `Add / Fix / Complete javadoc` commits.

Reply:
Hi @ok2c 

my idea it's progressively add the missing javadoc. I am not sure whether to do it in several PRs or in just one

Reply:
@arturobernalg To make my life a bit easier could you please keep those changes on a private batch and merge that branch with a single PR once ready?

Reply:
:) you're right. I apologize.